docs/optimizing/advanced_retrieval/How to get +15% hit_rate improvement using DeepMemory.ipynb

```diff
@@ -0,0 +1,749 @@
+{
```

Comment:
<p>the results look great.</p><p>probably can add more metrics like precision, MAP, NDCG in future with different top_k, where k at 5, 15, 20 so on</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/8625/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Reply:
thanks a lot for suggestions, we will definitely add them later